frontend/src/metabase/visualizations/shared/hooks/use-two-dimensional-chart-series.ts

```diff
@@ -0,0 +1,45 @@
+import { useMemo } from "react";
+import { DatasetData, VisualizationSettings } from "metabase-types/api";
+import { getChartColumns } from "metabase/visualizations/lib/graph/columns";
+import { ColumnFormatter } from "metabase/visualizations/shared/types/format";
+import { getSeries } from "../utils/data";
+import { getSeriesColors } from "../utils/colors";
+import { TwoDimensionalChartData } from "../types/data";
+
+export const useTwoDimensionalChartSeries = (
```

Comment:
Should it be just a utility method? It doesn't seem that memoizing individual functions instead of the whole one gives any benefits. Or does it?